ios/FluentUI/Date Time Pickers/Date Picker/DatePickerController.swift

```diff
@@ -246,21 +245,6 @@ class DatePickerController: UIViewController, GenericDateTimePicker {
         }
         self.segmentedControl = segmentedControl
     }
-=======
```

Comment:
i am surprised by this change? because tokenize segmented control got merged to mainbranch, what main has should be correct?

Reply:
I do have what is in main branch currently. The spacing was different for some reason, maybe that's the change you're seeing?

What do you suggest changing here? I'm not sure there is anything to do.

Reply:
Hmm it is no longer showinf the weird diff after the latest commit